php/class-fieldmanager-zone-field.php

```diff
@@ -202,6 +214,23 @@ public function presave( $values, $current_values = array() ) {
 			}
 		}
 
+		/**
+		 * Alter values before they go through the sanitize & save routine.
+		 *
+		 * Here, we're enforcing $post_limit.
+		 *
+		 * @param  array $values Field values being saved.
+		 * @param  array $current_values Field's previous values.
+		 * @return array Altered values.
+		 */
+		public function presave_alter_values( $values, $current_values = array() ) {
```

Comment:
This class is an extension of `Fieldmanager_Field` (that should answer both questions). I _should_ have called `parent::presave_alter_values()`, so I updated it to do that.